nan_json.h

```diff
@@ -77,15 +77,19 @@ class JSON {
 #if NAN_JSON_H_NEED_PARSE
     return scope.Escape(parse(json_string));
 #else
+    Nan::MaybeLocal<v8::Value> result;
+#if NODE_MODULE_VERSION == 14  // v0.12
```

Comment:
You can cherry-pick it from me if you like:  https://github.com/mkrufky/nan/commit/2b9bc1f827b0551d55028a39de1977262d1ca04e